### DISCUSSION TITLE
Saved map points information format in PODs

### DISCUSSION THREAD

**Discussion by Pelayo-Reguera** [Arquisoft/lomap_en3a#50] [Ideas]
Hi everyone!
I was thinking about storing all the information in a RDF format, so each type of lacalization like shops, restaurants, parks, etc, are stored in different solid data set (like different files) and in each data set all corresponding places. In addition, to devide them from public, friends or private, I thought on copy the same structure of files and placing them in the public and private directories, and for the ones to be shared only with friends a dedicated folder for them but inside the private one.

**Comment by Davidmc07** [comment]
I agree with using RDF over JSON. However, it may be better to store the categories as attributes of the place and to create a "LoMap" folder at the same level as public and private directories. In this case, we could change the permissions of its subfolders.

<img src="https://user-images.githubusercontent.com/98897872/224793732-fc69625c-8460-4ef6-afc5-a2b9d527034d.PNG" width="70%" />

**Reply by Pelayo-Reguera** [reply]
I don't get the point of creating another folder called LoMap because the permissions are still controlled by the user from its POD management webpage. I mean, everything is equals to what I said but inside a big folder called "LoMap". I think...

**Reply by Davidmc07** [reply]
There should be a method in the inrupt API to change the permissions of each folder. By creating the LoMap folder you avoid having your data spread across multiple directories and mixed with other stuff that does not belong to the application. In addittion, it is easier to organize and you may also have information that needs a different privacy level than the one that comes by default in the private and public folders.

**Comment by Davidmc07** [comment]
We could also create folders for shared maps in the group creator's POD and make the rest of the users editors .
